evm/src/main/java/org/hyperledger/besu/evm/worldstate/UpdateTrackingAccount.java

```diff
@@ -148,6 +148,11 @@ public Map<UInt256, UInt256> getUpdatedStorage() {
     return updatedStorage;
   }
 
+  @Override
+  public boolean isNewAccount() {
+    return account == null;
```

Comment:
Same here, not clear why this means it's a new account.

For example, https://github.com/hyperledger/besu/pull/5430/files#diff-7f0493f7e9a429807ad4302ee3c0d24aaae4bebcb4ee73f9e2148db4b50d80bbR147

```
return new WrappedEvmAccount(track(new UpdateTrackingAccount<>(newAccount)))
```

we create a new account and pass it to UpdateTrackingAccount constructor, but this isNewAccount would return false.

Reply:
Ok, in the cases where the new account is mutable I changed the code to delegate to that account for newness checking, otherwise it require absence of a parent to be the indicator for newness.